examples/nvme/perf/perf.c

```diff
@@ -1243,7 +1259,7 @@ print_performance(void)
 		worker = worker->next;
 	}
 
-	printf("========================================================\n");
+	printf("==========================================================================\n");
```

Comment:
I don't think we need to add additional `=` characters. Let's remove them and add if someone asks to add them during review in upstream.